org.idempiere.webservices/WEB-INF/src/org/idempiere/adinterface/ModelADServiceImpl.java

```diff
@@ -1213,6 +1209,9 @@ public StandardResponseDocument scanFields(DataField[] fields,MWebServiceType m_
 								+ ": input column " + field.getColumn() + " does not exist");
 					} else {
 						try {
+							if(!fieldInput.getAD_Column().isAlwaysUpdateable() && isProcessed)
+								throw new IdempiereServiceFault("Document Processed", new QName("ProcessedDocument"));

```

Comment:
This will change the existing error response. What will be the new error response ?

Reply:
the result will be like this : 

{
    "StandardResponse": {
        "@RecordID": 103,
        "@IsError": true,
        "Error": "Web service type UpdateInvoice: input column DocumentNo value could not be set: Document Processed"
    }
}

i tried update DocumentNo column on completed invoice